File: nodejs/sellect.js
if (require.main === module) {
  main()
}

function select(arr) {
  for (let i = 0; i < arr.length; i++) {
    let index = -1
    let min = 9999999999
    for (let j = i; j < arr.length; j++) {
      if (arr[j] < min) {
        index = j
        min = arr[j]
      }
    }
    tem = arr[index]
    arr[index] = arr[i]
    arr[i] = tem
  }
  return arr
}

function main() {
  ram = []
  for (let i = 0; i < 10; i++) {
    ram[i] = Math.floor(Math.random() * 100)
  }
  console.log(ram)
  console.log(select(ram))
}

File: python/sellect.py
def sellect(arr):
  for i in range(len(arr)):
    _min = 9999999
    index = -1
    for j in range(i,len(arr)):
      if arr[j] < _min:
        index = j
        _min = arr[j]
    arr[i],arr[index] = arr[index],arr[i]
  return arr  

import random

ram = [random.randint(0,1000) for i in range(100)]

print(sellect(ram))